### PR TITLE
examples: added svelte integration

### LINKED ISSUE #149: Integration: svelte
**Context**
To make it easier to start jitar using the frameworks developers are known with, we need to create integrations and examples for the most popular frameworks.

**Solution**
To support developers that use [svelte](https://svelte.dev/) we need a plugin and an example how to use it with svelte.

### PR DESCRIPTION
**Pull request by BasMasking**
Fixes #149 

Changes proposed in this pull request:
- Build the svelte integration example

@MaskingTechnology/jitar

### CODE REVIEW

File: examples/4-integrations/3-svelte/src/lib/Counter.svelte [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<script lang="ts">
```
Comment (petermasking) [inline examples/4-integrations/3-svelte/src/lib/Counter.svelte:1]: It seems that this file is not being used. Can it be removed?

File: examples/4-integrations/3-svelte/src/App.svelte [hunk 2]
```diff
@@ -0,0 +1,42 @@
+<script lang="ts">
+  import svelteLogo from './assets/svelte.svg'
```
Comment (petermasking) [inline examples/4-integrations/3-svelte/src/App.svelte:2]: Start with an empty line?

File: examples/4-integrations/3-svelte/src/vite-env.d.ts [hunk 3]
```diff
@@ -0,0 +1,2 @@
+/// <reference types="svelte" />
```
Comment (petermasking) [inline examples/4-integrations/3-svelte/src/vite-env.d.ts:1]: Is this file used? I don't have it when I create a new project using Vite.

File: package.json [hunk 4]
```diff
@@ -23,7 +23,8 @@
         "examples/3-apps/1-full-stack",
         "examples/3-apps/2-microservices",
         "examples/4-integrations/1-react",
-        "examples/4-integrations/2-vue"
+        "examples/4-integrations/2-vue",
+        "examples/4-integrations/3-svelte"
```
Comment (petermasking) [inline package.json:27]: The Svelte example is missing in the `examples/README.md` file.

**Review by petermasking — CHANGES_REQUESTED** [review]

File: examples/4-integrations/3-svelte/src/lib/Counter.svelte (hunk 1 above)
Comment (BasMasking) [reply examples/4-integrations/3-svelte/src/lib/Counter.svelte:1]: Removed

File: examples/4-integrations/3-svelte/src/App.svelte (hunk 2 above)
Comment (BasMasking) [reply examples/4-integrations/3-svelte/src/App.svelte:2]: It didn't, now it does :)

File: examples/4-integrations/3-svelte/src/vite-env.d.ts (hunk 3 above)
Comment (BasMasking) [reply examples/4-integrations/3-svelte/src/vite-env.d.ts:1]: I do get these with the generation. But I have no idea what it does. Seems like IntelliSense definitions.
https://v2.vitejs.dev/guide/env-and-mode.html#env-files

File: package.json (hunk 4 above)
Comment (BasMasking) [reply package.json:27]: Added to the read me

File: examples/4-integrations/3-svelte/src/App.svelte [hunk 5]
```diff
@@ -0,0 +1,43 @@
+
```
Comment (petermasking) [inline examples/4-integrations/3-svelte/src/App.svelte:1]: I didn't expect the new line here, but after the script open tag.

**Review by petermasking — CHANGES_REQUESTED** [review]

**Review by petermasking — DISMISSED** [review]

**Review by petermasking — APPROVED** [review]

**Pull request merged**